src/Resolvers/Microsoft.NET.Sdk.WorkloadManifestReader/SdkFeatureBand.cs

```diff
@@ -7,7 +7,11 @@
 #nullable disable
 namespace Microsoft.NET.Sdk.WorkloadManifestReader
 {
+#if SDK_FEATURE_BAND_INTERNAL
+    internal struct SdkFeatureBand : IEquatable<SdkFeatureBand>, IComparable<SdkFeatureBand>
+#else
     public struct SdkFeatureBand : IEquatable<SdkFeatureBand>, IComparable<SdkFeatureBand>
+#endif
```

Comment:
What do you think about something like this, to avoid the duplication?

```c#
#if SDK_FEATURE_BAND_INTERNAL
    internal
#else
    public
#endif
    struct SdkFeatureBand : IEquatable<SdkFeatureBand>, IComparable<SdkFeatureBand>
```